Hm, can you try rebasing?

```
19:05:49 # github.com/docker/docker/cli/command/formatter
19:05:49 cli/command/formatter/history_test.go:13:2: cannot find package "github.com/docker/docker/pkg/testutil/assert" in any of:
19:05:49 	/go/src/github.com/docker/docker/vendor/github.com/docker/docker/pkg/testutil/assert (vendor tree)
19:05:49 	/usr/local/go/src/github.com/docker/docker/pkg/testutil/assert (from $GOROOT)
19:05:49 	/go/src/github.com/docker/docker/pkg/testutil/assert (from $GOPATH)
19:05:49 FAIL	github.com/docker/docker/cli/command/formatter [setup failed]
```

Not related to this PR, but a package was changed today I think. Thought it was fixed already, but possibly not

--- a/daemon/daemon.go
+++ b/daemon/daemon.go
@@ -955,7 +955,7 @@ func prepareTempDir(rootDir string, rootUID, rootGID int) (string, error) {
 	if tmpDir = os.Getenv("DOCKER_TMPDIR"); tmpDir == "" {
 		tmpDir = filepath.Join(rootDir, "tmp")
 		newName := tmpDir + "-old"
-		if err := os.Rename(tmpDir, newName); err != nil {
+		if err := os.Rename(tmpDir, newName); err == nil {
 			go func() {
 				if err := os.RemoveAll(newName); err != nil {
 					logrus.Warnf("failed to delete old tmp directory: %s", newName)

